src/Umbraco.Web.UI.Client/src/common/services/filemanager.service.js

```diff
@@ -14,6 +14,17 @@ function fileManager($rootScope) {
 
 
     var mgr = {
+        /**
+         * @ngdoc property
+         * @name umbraco.services.fileManager#maxFileSize
+         * @propertyOf umbraco.services.fileManager
+         * @type {Number}
+         * @default 0
+         * @description
+         * The max file size allowed to be uploaded to the server in bytes
+         */
+        maxFileSize: parseInt(Umbraco.Sys.ServerVariables.umbracoSettings.maxFileSize ?? '0', 10),
```

Comment:
We are missing the multiplication here, as we had it before: `Number(Umbraco.Sys.ServerVariables.umbracoSettings.maxFileSize ?? 0) * 1000`